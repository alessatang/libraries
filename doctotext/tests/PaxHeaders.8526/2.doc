30 atime=1389151462.464930405
30 ctime=1389151462.464930405
