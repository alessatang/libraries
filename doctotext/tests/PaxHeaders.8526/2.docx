29 atime=1389151462.47393037
29 ctime=1389151462.47393037
