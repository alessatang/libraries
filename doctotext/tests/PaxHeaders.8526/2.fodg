30 mtime=1384978086.716497429
30 atime=1389151462.489930307
30 ctime=1389151462.489930307
