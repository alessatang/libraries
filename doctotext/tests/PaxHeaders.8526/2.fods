30 mtime=1384978086.738497437
30 atime=1389151463.673925618
30 ctime=1389151463.673925618
