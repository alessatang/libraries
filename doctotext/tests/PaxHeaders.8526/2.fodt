30 mtime=1384978086.739497437
30 atime=1389151463.673925618
30 ctime=1389151463.673925618
