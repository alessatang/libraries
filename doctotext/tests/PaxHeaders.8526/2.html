30 atime=1389151463.673925618
30 ctime=1389151463.673925618
