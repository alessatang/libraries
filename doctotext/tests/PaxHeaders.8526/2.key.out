30 mtime=1379553520.866579823
30 atime=1389151465.189919589
30 ctime=1389151465.189919589
